disperser/dataapi/server.go

```diff
@@ -465,7 +465,9 @@ func (s *server) FetchNonSigners(c *gin.Context) {
 //	@Summary	Fetch operators non signing percentage
 //	@Tags		Metrics
 //	@Produce	json
-//	@Param		interval	query		int	false	"Interval to query for operators nonsigning percentage [default: 3600]"
+//	@Param		interval	query		int		false	"Interval to query for operators nonsigning percentage [default: 3600]"
```

Comment:
It may be simpler to just have 2 params: endtime and interval. When endtime is missing, set it to time.Now(). Then it'll be also be backward compatible.

Reply:
That seems like a good suggestion. I'll go ahead and update.